src/mongo/crypto/crypto_openssl.cpp

```diff
@@ -39,25 +39,47 @@
 #include <openssl/hmac.h>
 #include <openssl/sha.h>
 
+#if OPENSSL_VERSION_NUMBER < 0x10100000L
```

Comment:
Please wrap these functions in an anonymous namespace.
IE:
namespace {
... the current code...
}  // namespace

It keeps these symbol names from potentially clashing with others in different translation units.